src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaItem.tsx

```diff
@@ -183,6 +210,10 @@ function SchemaItem(props: SchemaItemProps) {
     if (itemToDisplay.properties) {
       items.push(renderProperties(itemToDisplay.properties));
     }
+    const combinationChildren = item.allOf || item.oneOf || item.anyOf;
```

Comment:
then this could simply be `item.combination`

Reply:
Jupp.